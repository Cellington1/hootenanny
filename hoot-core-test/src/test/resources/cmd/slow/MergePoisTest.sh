#!/bin/bash
set -e

/usr/bin/node $HOOT_HOME/hoot-core-test/src/test/resources/cmd/slow/MergePoisTest.js 
