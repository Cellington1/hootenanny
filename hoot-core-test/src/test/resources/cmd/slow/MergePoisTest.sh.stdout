<?xml version="1.0" encoding="UTF-8"?>
<osm version="0.6" generator="hootenanny" srs="GEOGCS[&quot;WGS 84&quot;,&#10;    DATUM[&quot;WGS_1984&quot;,&#10;        SPHEROID[&quot;WGS 84&quot;,6378137,298.257223563,&#10;            AUTHORITY[&quot;EPSG&quot;,&quot;7030&quot;]],&#10;        TOWGS84[0,0,0,0,0,0,0],&#10;        AUTHORITY[&quot;EPSG&quot;,&quot;6326&quot;]],&#10;    PRIMEM[&quot;Greenwich&quot;,0,&#10;        AUTHORITY[&quot;EPSG&quot;,&quot;8901&quot;]],&#10;    UNIT[&quot;degree&quot;,0.0174532925199433,&#10;        AUTHORITY[&quot;EPSG&quot;,&quot;9108&quot;]],&#10;    AUTHORITY[&quot;EPSG&quot;,&quot;4326&quot;]]">
    <node visible="true" id="-2" timestamp="1970-01-01T00:00:00Z" version="1" lat="48.0485400000000027" lon="11.7019400000000005">
        <tag k="hoot:status" v="2"/>
        <tag k="name" v="POST, JÃ„GER-VON-FALL-STRASSE"/>
        <tag k="amenity" v="post_office"/>
        <tag k="error:circular" v="150"/>
    </node>
</osm>

<?xml version="1.0" encoding="UTF-8"?>
<osm version="0.6" generator="hootenanny" srs="GEOGCS[&quot;WGS 84&quot;,&#10;    DATUM[&quot;WGS_1984&quot;,&#10;        SPHEROID[&quot;WGS 84&quot;,6378137,298.257223563,&#10;            AUTHORITY[&quot;EPSG&quot;,&quot;7030&quot;]],&#10;        TOWGS84[0,0,0,0,0,0,0],&#10;        AUTHORITY[&quot;EPSG&quot;,&quot;6326&quot;]],&#10;    PRIMEM[&quot;Greenwich&quot;,0,&#10;        AUTHORITY[&quot;EPSG&quot;,&quot;8901&quot;]],&#10;    UNIT[&quot;degree&quot;,0.0174532925199433,&#10;        AUTHORITY[&quot;EPSG&quot;,&quot;9108&quot;]],&#10;    AUTHORITY[&quot;EPSG&quot;,&quot;4326&quot;]]">
    <node visible="true" id="-7" timestamp="1970-01-01T00:00:00Z" version="1" lat="12.3456782000000000" lon="23.4567891000000017">
        <tag k="poi" v="yes"/>
        <tag k="hoot:status" v="1"/>
        <tag k="alt_name" v="Test post office"/>
        <tag k="name" v="Test post office - More Complete"/>
        <tag k="amenity" v="post_office"/>
        <tag k="error:circular" v="2"/>
    </node>
</osm>

